docs/getting-started/typescript-options-api.md

```diff
@@ -11,19 +11,13 @@ position: 30
 
 # Get Started with Kendo UI for Vue
```

Comment:
See the comments in `docs/getting-started/javascript-options-api.md` as they are valid for this file as well.